policy/modules/kernel/filesystem.te

```diff
@@ -138,6 +138,7 @@ fs_type(nfsd_fs_t)
 genfscon nfsd / gen_context(system_u:object_r:nfsd_fs_t,s0)
 
 type nsfs_t;
+files_type(nsfs_t)
```

Comment:
I don't think this is necessary. There should be no nsfs_t file outside of nsfs. Am I missing something?